src/cdk/text-field/_text-field.scss

```diff
@@ -8,11 +8,13 @@
   @keyframes cdk-text-field-autofill-end {/*!*/}
 
   .cdk-text-field-autofill-monitored:-webkit-autofill {
-    animation-name: cdk-text-field-autofill-start;
+    // In Chrome 80 we need a 1ms delay, or the animationstart event won't fire.
```

Comment:
Maybe we should say "Since Chrome 80"? Otherwise it sounds like it's only necessary for Chrome 80 specifically.